### PR TITLE
Add extractor arg for Hotstar to ignore tags

### PR DESCRIPTION
**Pull request by Ashish0804**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [ ] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [ ] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [ ] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Usage:  `--extractor-args 'hotstar:res=sd,hd;vcodec=h264;dr=sdr'`

### CODE REVIEW

**Comment by Ashish0804** [comment]
@pukkandan why add pending fixes?
What do u want me to fix? 🤔

**Comment by pukkandan** [comment]
add the fields to readme. Thought we discussed this on discord

File: README.md [hunk 1]
```diff
@@ -1621,6 +1621,11 @@ The following extractors use this feature:
 #### gamejolt
 * `comment_sort`: `hot` (default), `you` (cookies needed), `top`, `new` - choose comment sorting mode (on GameJolt's side)
 
+#### hotstar
+* `res`: resolution to ignore - one or more of `sd`, `hd`, `fhd`
```
Comment (pukkandan) [inline README.md:1625]: shouldn't it be `ignore-res` etc?

Comment (Ashish0804) [reply README.md:1625]: Tooooo long to type. More importantly... extractor args is for power users anyways ( explains why they are documented in the second half of the readme) so naming shouldn't be a concern as it's documented properly.

Comment (nao20010128nao) [reply README.md:1625]: _suggestion: `igres`_

**Pull request merged**